alphagov/paas-cf
# [#118836855] Revert "[TEMP] add IPs"

## What

This reverts commit 3772aa3.

This was added while reviewing #252 and shouldn't have been merged to the
master branch. It opened up our development environments for:

- anyone to access deployed apps
- a single EC2 instance to access the API

Before, when accessing from my personal VPS outside the office:

    dan@vps:~$ curl -km5 https://test.dcarley.dev.cloudpipelineapps.digital/
    404 Not Found: Requested route ('test.dcarley.dev.cloudpipelineapps.digital') does not exist.

After:

    dan@vps:~$ curl -km5 https://test.dcarley.dev.cloudpipelineapps.digital/
    curl: (7) Failed to connect to test.dcarley.dev.cloudpipelineapps.digital port 443: Connection timed out

## How to review

1. Deploy an environment from this branch.
1. Confirm that the pipeline is green.
1. Confirm, if possible, that you can't connect to a deployed app from outside the office.

## Who can review

Not @dcarley
